app/controllers/api/base_controller/parser.rb

```diff
@@ -72,7 +72,7 @@ def validate_api_parameters
       def parse_href(href)
         if href
           path = href.match(/^http/) ? URI.parse(href).path.sub(/\/*$/, '') : href
-          path = "#{@prefix}/#{path}" unless path.match(@prefix)
+          path = "/api/#{path}" unless path.match("/api")
```

Comment:
Let's leverage "/#{Settings.base.module}" instead of /api, this was not hard coded before.  Thanks.